.ci/linux-steps.yaml

```diff
@@ -12,7 +12,7 @@ steps:
     tar -xvzpf v1.3.0.tar.gz # Unpack into cereal-1.3.0/.
 
     git clone --depth 1 https://github.com/mlpack/jenkins-conf.git conf
-    git clone --depth 1 https://github.com/mlpack/mlpack.git
+    git clone --depth 1 -b depthwise https://github.com/Aakash-kaushik/mlpack.git
```

Comment:
Can you add a comment above, that explains why we build against your fork for now.